core/client/fs/src/main/java/alluxio/client/block/stream/NettyPacketWriter.java

```diff
@@ -481,13 +489,13 @@ public void operationComplete(ChannelFuture future) {
   }
 
   /**
-   * The netty channel future listener that is called when a EOF or CANCEL is complete.
+   * The netty channel future listener that is called when a EOF or CANCEL or FLUSH is complete.
    */
-  private final class EofOrCancelListener implements ChannelFutureListener {
+  private final class EofOrCancelorFlushListener implements ChannelFutureListener {
```

Comment:
This listener cannot be reused because `Cancel` and `EOF` are terminal RPCs whereas `flush` is not

Reply:
Thanks for the kind remind. Add a new `FlushListener` for flush only.